JobConfig/pileup/prolog.fcl

```diff
@@ -22,6 +22,7 @@ Pileup: {
       module_type : CompressDetStepMCs
       strawGasStepTag : "StrawGasStepMaker"
       caloShowerStepTag : "CaloShowerStepMaker"
+      surfaceStepTag : ""
```

Comment:
Thanks, Dave. I think this should be ```surfaceStepTags : [ ]```

Reply:
Hmm, it worked fine when I tested it. The code seems to require an atom, not a sequence. Maybe you're thinking of CompressDigiMCS?